### DISCUSSION TITLE
Ajuste após criação

### DISCUSSION THREAD

**Discussion by metantropia** [YuriAlessandro/gerador-ficha-tormenta20#307] [Ideias]
Se fosse possivel ajustar valores e escolhas depois da criação seria ótimo, mas como sei que é um baita desafio, exportar para um excel, pdf editável já seria muito da hora. Parabens pelo projeto, ficou muito top.

**Comment by YuriAlessandro** [comment]
Mudar a ficha após ela ser criada é algo que não passa pela nossa cabeça. A ideia é gerar uma ficha aleatória mesmo. Mas a questão de exportação é algo que queremos fazer mais para a frente.

**Reply by metantropia** [reply]
Entendo, e faz sentido, vira uma demanda enorme de criação dessa plataforma para fazer as alterações, mas só a exportação para outras plataformas que fazem isso já seria ótimo, tipo excel, Orc n' Roll, Roll20 e etc.
